src/runtime/cpu/loss/loss.hpp

```diff
@@ -0,0 +1,32 @@
+#include "runtime/cpu/kernels/kernels.hpp"
+
+namespace deepworks {
```

Comment:
I would add `namespace losses`  as it done for `metrics`

Reply:
It seems it's my mistake, it should be `loss`, but it doesn't matter in general.
https://github.com/NeuralDeepWorks/deepworks/blob/main/include/deepworks/metrics.hpp#L6